src/common/Network/PacketDef/Ipcs.h

```diff
@@ -182,6 +182,7 @@ namespace Sapphire::Network::Packets
     Mount = 0x01E8, // updated 4.5
 
     DirectorVars = 0x01EA, // updated 4.5
+    DirectorPopUp = 0x01F5, // display dialogue pop-ups in duties and FATEs, for example, Teraflare's countdown
```

Comment:
`DirectorPopUp` isn't right I don't think, there's an exd with all of these in there from my knowledge so it should be called that, but in any case I don't think it has any relation to directors

Reply:
ah yeah all in `InstanceContentTextData` so it could be related to directors but not sure, will need to suss out the packet

Reply:
I see. I thought it was related to directors because the packet I captured didn’t do anything unless it was injected after sending the correct DirectorInit for it.

Reply:
What are the parameters used in this? Also i noticed that it may not be pulling just from that table but also from quest exd tables.

Reply:
We've just looked into it, It really appears to be a DirectorMessage packet.